src/components/20-molecules/datepicker/story.js

```diff
@@ -55,7 +55,7 @@ story.add('Datepicker', () => {
   const wrapper = document.createElement('div');
 
   const template = html`
-    <div style="background-color: lightgray">
+    <div style="background-color: lightgrey">
```

Comment:
This `lightgrey` is the build-in css variable. Just to be sure you do not make a replace mistake. but i think its ok

Reply:
This was on purpose. Shouldn't make a difference, due the fact that both have the same color code.